appveyor.yml

```diff
@@ -46,6 +46,15 @@ test_script:
       $Env:PATH = "C:\${Env:PLATFORM}\bin;${Env:PATH}"
       $Env:LOG_SPEC_ORDER = 'true'
       Get-ChildItem Env: | % { Write-Output "$($_.Key): $($_.Value)"  }
+      if ( $(ruby --version) -match "2\.4" ) {
```

Comment:
Almost.

In this case the `ruby --version` string is something like `ruby 2.4.0p0 (2016-12-24 revision 57164) [x86_64-darwin15]`... so I could change it to:

```powershell
if ( $(ruby --version) -match "^ruby\s+2\.4" ) {
```